docs/iris/src/whatsnew/contributions_1.11/bugfix_2016-Oct-06_is_circular.txt

```diff
@@ -0,0 +1,2 @@
+* Fixed a bug where :func:iris.util._is_circular would erroneously return False
```

Comment:
The markup here is no longer correct, it is missing the back-ticks ( I suspect because copy-paste from the Github comment, which turned the back-ticks into an `object`). You actually don't need to capitalise False in this usage unless you are referring explicitly to the Python object `False`...